app/views/notifications/_notification.html.erb

```diff
@@ -0,0 +1 @@
+<%= notification.message %> Look at that -  <%= link_to_subject notification.subject %><br>
```

Comment:
It's much more controllable to wrap each notification item in `<p>` or `<div>`. Plain text with `<br>` tags will really bring you tears when you'll try to apply CSS to it.